src/Components/SettingsPage/SettingsPage.css

```diff
@@ -39,6 +39,22 @@
     cursor: pointer;
 }
 
+.settingsPage .actionButtons {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    gap: 1rem;
+}
+
+.settingsPage .actionButtons button {
+    font-size: 1rem;
+    padding: 0.5rem;
+    border: none;
+    border-radius: 5px;
+    background: #f0f0f0;
+    cursor: pointer;
+}
```

Comment:
could you add some more pliancy on the buttons? Great with the pointer, but should have some highlight as well :)